addons/mail/tests/test_mail_composer.py

```diff
@@ -41,181 +41,6 @@ def setUpClass(cls):
-            'Default value is kept at save'
-        )
-
-        with self.mock_mail_gateway():
-            saved_form._action_send_mail()
-
-        message = self.test_record.message_ids[0]
-        self.assertEqual(message.body, '<p>Hello</p>')
-        self.assertEqual(message.partner_ids, partner_classic)
-        self.assertEqual(message.subject, f'{test_record.name}')
-
-    @mute_logger('odoo.addons.mail.models.mail_mail')
-    @users('employee')
-    def test_composer_default_recipients_private(self):
-        """ Test usage of a private partner in composer, as default value """
-        partner_private = self.partner_private.with_env(self.env)
-        partner_classic = self.partner_classic.with_env(self.env)
-        test_record = self.test_record.with_env(self.env)
-
-        form = Form(self.env['mail.compose.message'].with_context({
-            'default_partner_ids': (partner_private + partner_classic).ids,
-            'default_model': test_record._name,
-            'default_res_ids': test_record.ids,
-        }))
-        form.body = '<p>Hello</p>'
-        self.assertEqual(
-            sorted(form.partner_ids._get_ids()),
-            sorted((partner_private + partner_classic).ids),
-            'Default populates the field'
-        )
-        saved_form = form.save()
-        self.assertEqual(
-            saved_form.partner_ids, partner_private + partner_classic,
-            'Default value is kept at save'
-        )
-
-        with self.mock_mail_gateway():
-            saved_form._action_send_mail()
-
-        message = self.test_record.message_ids[0]
-        self.assertEqual(message.body, '<p>Hello</p>')
-        self.assertEqual(message.partner_ids, partner_private + partner_classic)
-        self.assertEqual(message.subject, f'{test_record.name}')
-
-    @mute_logger('odoo.addons.base.models.ir_rule', 'odoo.addons.mail.models.mail_mail')
-    @users('employee')
-    def test_composer_default_recipients_private_norights(self):
-        """ Test usage of a private partner in composer when not having the
-        rights to see them, as default value """
-        self.user_employee.write({'groups_id': [
-            (3, self.env.ref('base.group_private_addresses').id),
-        ]})
-        with self.assertRaises(AccessError):
-            _name = self.partner_private.with_env(self.env).name
-
-        partner_classic = self.partner_classic.with_env(self.env)
-        test_record = self.test_record.with_env(self.env)
-
-        with self.assertRaises(AccessError):
-            _form = Form(self.env['mail.compose.message'].with_context({
-                'default_partner_ids': (self.partner_private + partner_classic).ids,
-                'default_model': test_record._name,
-                'default_res_ids': test_record.ids,
-            }))
-
-    @mute_logger('odoo.addons.mail.models.mail_mail')
-    @users('employee')
-    def test_composer_template_recipients_private(self):
```

Comment:
Same here, could be interesting to migrate it to "the template contains reference to partners you can't access".